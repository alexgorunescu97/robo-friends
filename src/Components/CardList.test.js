import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'Test user',
            email: 'test.user@gmail.com'
        }
    ]
    expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
});