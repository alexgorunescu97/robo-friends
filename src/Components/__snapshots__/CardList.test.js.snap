// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <Card
    email="test.user@gmail.com"
    id={1}
    key="1"
    name="Test user"
  />
</div>
`;
